packages/core/helper-plugin/lib/src/components/GenericInput/utils/getFieldUnits.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {String} type
+ * @param {Number} minimum
+ * @param {Number} maximum
+ * @returns {'' | 'characters' | 'character'}
+ */
+const getFieldUnits = ({ type, minimum, maximum }) => {
+  if (type === 'number') {
+    return '';
+  }
+  const plural = Math.max(minimum || 0, maximum || 0) > 1;
+
+  if (plural) {
+    return 'characters';
+  }
+
+  return 'character';
```

Comment:
This needs to be a translateable string too. You should be able to use https://formatjs.io/docs/core-concepts/icu-syntax#plural-format here too.